setup.cfg

```
@@ -26,6 +26,7 @@ package_dir =
 packages = find:
 include_package_data = true
 zip_safe = false
+python_requires = >=2.7, !=3.0, !=3.1, !=3.2, !=3.3, !=3.4, !=3.5, !=3.6
```

```suggestion
python_requires = >=3.7
```

We don't really need to worry about Python 2 support now. 8 years of extended support from volunteers is more than enough.

---

Readability nit: I'd add a whitespace between the comparison operator and the version number if you don't mind.

```suggestion
python_requires = >= 3.7
```